modules/tensor_mechanics/src/materials/ComputeElasticityTensor.C

```diff
@@ -39,11 +39,17 @@ ComputeElasticityTensorTempl<is_ad>::ComputeElasticityTensorTempl(
     issueGuarantee(_elasticity_tensor_name, Guarantee::ISOTROPIC);
   else
   {
-    // Define a rotation according to Euler angle parameters
-    RotationTensor R(_Euler_angles); // R type: RealTensorValue
+    // Use user-provided rotation matrix if given
+    if (parameters.isParamValid("rotation_matrix"))
```

Comment:
This is a parameter of the `ComputeRotatedElasticityTensorBaseTempl` class. I think I'll refactor this in the future to have a rotate call in that class so that the details of _how_ the tensor is rotated is not exposed to the derived classes.

Reply:
Same thing for `_Euler_angles`

Reply:
> This is a parameter of the `ComputeRotatedElasticityTensorBaseTempl` class. 

Same goes for Euler angles

Reply:
Oh nvm, the parameter. Gotcha.